resources/assets/less/components/gistlog.less

```diff
@@ -28,6 +32,38 @@
     }
 }
 
+.gistlog-article {
+    margin-bottom: 4em;
+}
+
+.gistlog-article__title {
+
```

Comment:
Will do. Thanks for the heads up. psr-2 and alphabetized css property names... check :white_check_mark: